velox/type/Filter.h

```diff
@@ -865,6 +868,102 @@ class BigintValuesUsingHashTable final : public Filter {
         values_(other.values_),
         sizeMask_(other.sizeMask_) {}
 
+  T min() const {
+    return min_;
+  }
+
+  T max() const {
+    return max_;
+  }
+
+  const std::vector<T>& values() const {
+    return values_;
+  }
+
+  const std::vector<T>& hashTable() const {
+    return hashTable_;
+  }
+
+  bool testInt(T value) const {
```

Comment:
move this to cpp file

Reply:
Thanks for the suggestion, this was in the cpp file earlier but it resulted in the linux-build and linux-adapters CI jobs to fail with function not found error. Moving the definition to the header fixed the error.

Reply:
You need to define testInt128, and testInt64 in the CPP file as well.